src/screens/admin/DocumentConfig.tsx

```
@@ -39,6 +40,39 @@ const DocumentConfig = () => {
+	
+	// --- State for document types ---
+	const [documentTypes, setDocumentTypes] = useState<string[]>([]);
+	const [isLoadingDocumentTypes, setIsLoadingDocumentTypes] = useState(true);
+
+	// --- Fetch document types from API ---
+	useEffect(() => {
+		const fetchDocumentTypes = async () => {
+			try {
+				setIsLoadingDocumentTypes(true);
+				const response = await getMapping('documentTypeConfiguration');
+				if (response?.data?.value?.documentType && Array.isArray(response.data.value.documentType)) {
```

ESLint

[error] Replace `response?.data?.value?.documentType·&&·Array.isArray(response.data.value.documentType)` with `⏎↹↹↹↹↹response?.data?.value?.documentType·&&⏎↹↹↹↹↹Array.isArray(response.data.value.documentType)⏎↹↹↹↹`

(prettier/prettier)